getpaid/utils.py

```diff
@@ -81,3 +82,16 @@ def qs_to_ordered_params(query_string):
             field = field.decode('utf8')
         params[field] = value
     return params
+
+
+def get_domain(request=None):
+    if request and request.META.get('HTTP_HOST'):
```

Comment:
Maybe we could have `GETPAID_SITE_DOMAIN`, but is it really needed? Why can't you use Site?